.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -2,185 +2,189 @@ name: Bug Report
+    - type: markdown
+      attributes:
+          value: |
+              ## Before Submitting your Issue
+              Our GitHub issues are meant exclusively for bug reports and feature requests. Usage questions may be closed without comment. If you have a question, please check to see if your question is answered in the FAQ for the library you are using:
+              - [@azure/msal-browser](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/FAQ.md)
+              - [@azure/msal-node](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-node/docs/faq.md)
+              - [@azure/msal-angular](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/docs/FAQ.md)
+              - [@azure/msal-react](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-react/FAQ.md)
+              - [@azure/msal or msal](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/msal-lts/lib/msal-core/docs/FAQ.md)
+
+              In the case your question is not answered in the FAQ, please verify your question has not been covered on [Stack Overflow](https://stackoverflow.com/questions/tagged/msal.js).
+              If you can't find a similar question on Stack Overflow, please create a new Stack Overflow thread using the tag `msal.js`.
+
+              If you have questions about timelines, please see our [roadmap](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/roadmap.md). Issues pertaining to timelines may be closed without comment.
+
+    - type: markdown
+      attributes:
+          value: |
+              ## Issue Details
+
+    - type: dropdown
+      attributes:
+          label: Core Library
+          options:
+              - "MSAL.js (@azure/msal-browser)"
+              - "MSAL Node (@azure/msal-node)"
+      validations:
+          required: true
+
+    - type: input
+      attributes:
+          label: Core Library Version
+          description: "Please enter the latest version this issue can be reproduced in."
+          placeholder: "3.1.0"
+      validations:
+          required: true
+
+    - type: dropdown
+      attributes:
+          label: Wrapper Library
+          options:
+              - "MSAL Angular (@azure/msal-angular)"
+              - "MSAL React (@azure/msal-react)"
+              - "MSAL Node Extensions (@azure/msal-node-extensions)"
+              - "Not Applicable"
+      validations:
+          required: true
+
+    - type: input
+      attributes:
+          label: Wrapper Library Version
+          description: "Please enter the latest version this issue can be reproduced in. If you are not using a wrapper library please enter 'None'"
+          placeholder: "2.0.0"
+      validations:
+          required: true
+
+    - type: dropdown
+      attributes:
+          label: Public or Confidential Client?
```

Comment:
do we need a link off to explain public or confidential client? I feel like we get a lot of mis-matches here.